src/main/java/org/opentripplanner/routing/edgetype/StreetEdge.java

```diff
@@ -1134,4 +1377,18 @@ public boolean containsCarNetwork(String carNetwork) {
         }
         return carNetworks.contains(carNetwork);
     }
+
+    public boolean addVehicleNetwork(String vehicleNetwork) {
+        if (vehicleNetworks == null) {
+            vehicleNetworks = new HashSet<>();
+        }
+        return vehicleNetworks.add(vehicleNetwork);
+    }
+
+    public boolean containsVehicleNetwork(String vehicleNetwork) {
+        if (vehicleNetworks == null){
```

Comment:
Hmmm, I think you're right. However, I think moving forward it may be best to always require a vehicle network to be associated with a rental. Also, this may be in need of a mode-dependent rental network in a future refactor. I may just keep it the way it is for now.